### PR TITLE
feat(dialog): make dialog full size on mobile

### PR DESCRIPTION
**Pull request by matmkian**
# General summary

# Screenshots

### CODE REVIEW

File: packages/lumx-core/src/scss/components/dialog/_index.scss [hunk 1]
```diff
@@ -51,22 +55,39 @@
...
+
+        #{$self}--is-hidden & {
+            @include mq($until: desktop) {
+                transform: translateY(100%);
+            }
+        }
+
+        @include mq($until: desktop) {
+            transition: transform 500ms cubic-bezier(0.23, 1, 0.32, 1);
+            animation: lumx-dialog-show-mobile 500ms cubic-bezier(0.23, 1, 0.32, 1);
```
Comment (hchevallier) [inline packages/lumx-core/src/scss/components/dialog/_index.scss:67]: Could we store the duration in a variable `$lumx-responsive-dialog-transition-duration`, it is useful to synchronize other animations with dialog opening/closing. (`$lumx-dialog-transition-duration` is used in Lumapps)

**Review by hchevallier — APPROVED** [review]
LGTM

**Comment by FlorentDinet** [comment]
Do we have a mobile breakpoint to use instead of tablet?

I think its a bit too much for confirm dialog on iPad for example:
Capture d’écran 2024-03-25 à 14 27 14

**Comment by FlorentDinet** [comment]
Ok for this change, we'll optimise later the breakpoint

**Pull request merged**